spec/models/movie_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Movie, type: :model do
+  describe '#validations' do
+    let(:movie) { build(:movie) }
+
+    it { is_expected.to validate_presence_of(:title) }
+    it { is_expected.to validate_presence_of(:description) }
+    it { is_expected.to validate_presence_of(:genre) }
+    it { is_expected.to validate_presence_of(:director) }
+    it { is_expected.to validate_presence_of(:duration) }
+    it { is_expected.to validate_numericality_of(:duration).is_greater_than(10) }
+  end
```

Comment:
I saw that you provided `shoulda-matchers` to the app. 
First of all, you can use `subject` instead of `let(:movie)` cause we can setup the subject of the example group for all those tests. 
When we will have the subject prepared we can use a method `should` from `shoulda-matchers`. 

```suggestion
  describe '#validations' do
    subject { build(:movie) }

    it { should validate_presence_of(:title) }
    it { should validate_presence_of(:description) }
    it { should validate_presence_of(:genre) }
    it { should validate_presence_of(:director) }
    it { should validate_presence_of(:duration) }
    it { should validate_numericality_of(:duration).is_greater_than(10) }
  end
```

btw. I don't think that your way of solving this problem is bad - completely not, it's pretty good, but wanted to show you another way with `shoulda-matchers` usage